tests/hypothesis_strategies/alternative_creation/test_utils.py

```diff
@@ -15,5 +15,12 @@
     ],
 )
 def test_none_bounds(lower, upper):
-    """Bounds can also be None."""
+    """Bounds can be None."""
     Interval(lower, upper)
+
+
+def test_deserialization_via_classmethod():
```

Comment:
ideally the test function names should express that these are about intervals (since the file is called utils hence implies a broader scope)

Reply:
Thanks, very good point. Instead of renaming the test function, I instead narrowed the scope by renaming the file (because the same issue applied also to the other function and putting everything in "utils" is probably not a great idea anyway).